tests/conftest.py

```diff
@@ -257,6 +259,17 @@ def _initialize_with_retries(
     pytest.fail("Failed to communicate with Trezor")
 
 
+def _skip_async_confirmation(raw_client: Client) -> None:
```

Comment:
not totally sure how this works
i assume you're trying to skip over the "show continue in app" screen, right?
this is going to be racey in some very weird cases though: because the dialog is on a timeout, you have a TOCTOU on the `press_yes`, which may end up going to the homescreen

i'd suggest `synchronize_at("Homescreen")` (maybe that will be a problem when it's RecoveryHomescreen?? dunno)
and on the Rust side, adding code that, if animations are disabled, ignore the timeout and display "continue in the app" for one frame only

Reply:
> i assume you're trying to skip over the "show continue in app" screen, right?

Yes - I was mainly trying to reduce the tests' duration, following https://github.com/trezor/trezor-firmware/pull/5097#issuecomment-2911128376.

> on the Rust side, adding code that, if animations are disabled, ignore the timeout and display "continue in the app" for one frame only

Good idea, thanks! It should definitely help :)